src/hotspot/share/interpreter/zero/bytecodeInterpreter.cpp

```diff
@@ -1365,19 +1365,19 @@ void BytecodeInterpreter::run(interpreterState istate) {
           CONTINUE;                                                          \
       }
 
-#define NULL_COMPARISON_NOT_OP(name)                                         \
+#define nullptr_COMPARISON_NOT_OP(name)                                         \
```

Comment:
These ones don't look right. I think this is the generic "null" forced into uppercase because it is a macro.

Reply:
Thanks David, nice catch!